templates/index.html

```diff
@@ -160,36 +160,25 @@
 
 
 
-  <paper-dialog id="about">
+  <paper-dialog class = "about-dialog" id="about">
     <div id="aboutpanel">
       <h2>About</h2>
       <div>EPSchedule Club: 
         <p>---A club designated to maintain this website and add changes</p>
         <p>---We meet in TMAC-007, every Thursday during Middle Band</p>
       </div>
-      <div>Club Leader: Kalie Uberti: 
-        <a href="mailto:kuberti@eastsideprep.org">kuberti@eastsideprep.org</a>
-        <p>---Club leader and manager of this project</p>
-      </div>
-      <div>Angad Josan: 
+      <div>Club Leaders: Angad Josan and Ryan Li:
         <a href="mailto:ajosan25@eastsideprep.org">ajosan25@eastsideprep.org</a>
-        <p>---</p>
-      </div>
-      <div>Ryan Li: 
         <a href="mailto:rli@eastsideprep.org">rli@eastsideprep.org</a>
-        <p>---Created this about page</p>
-      </div>
-      <div>Andrei Espelien: 
-        <a href="mailto:aespelien@eastsideprep.org">aespelien@eastsideprep.org</a>
-        <p>---Mediocre competitive programmer</p>
+        <p>---Club leaders and managers of this project</p>
       </div>
-      <div>Seamus Fu: 
-        <a href="mailto:sfu@eastsideprep.org">sfu@eastsideprep.org</a>
-        <p>---</p>
+      <div>Jack Goetzmann: 
+        <a href="mailto:jgoetzmann@eastsideprep.org">jgoetzmann@eastsideprep.org</a>
+        <p>---Added commenting and work on the front end</p>
```

Comment:
These descriptions may get out of date, maybe we just want a list of club members here?